src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -80,7 +80,9 @@
     (let [anim-opacity (animation/create-value 0)]
       [react/animated-view {:style (style/message-timestamp-wrapper message) :opacity anim-opacity}
        (when @show-timestamp? (message-timestamp-anim anim-opacity show-timestamp?))
-       [react/text {:style (style/message-timestamp-text)}
+       [react/text
+        {:style               (style/message-timestamp-text)
+         :accessibility-label :message-timestampt}
```

Comment:
typo 😅 `timestampt`

Reply:
Yeah... thanks! I blindly copied it form the issue description. On the other hand, everything is implemented according to requirements :D